main.py

```diff
@@ -67,25 +80,16 @@ def log_crash(logtype, value, tb):
     print('')
 
 
-# Version Number to be displayed.
-# This will only be shown as a fallback, when the git commit hash can't be found.
-VERSION_NUMBER = "Ver. 0.7.0dev"
-
-
-if os.path.exists("commit.txt"):
-    with open("commit.txt", 'r', encoding='utf-8') as read_file:
-        print("Running on pyinstaller build")
-        VERSION_NUMBER = read_file.read()
-else:
+if get_version_info().is_source_build:
```

Comment:
yeah please store the output of get_version_info() bc it looks cluttered calling the function this many times

Reply:
The function actually statically stores the VersionInfo instance anyways to avoid redundant disk access, so I guess it's just a matter of preference. There shouldn't be any overhead.

Reply:
oooh thats actually really smart, props